### PR TITLE
[Feature][Elasticsearch] Support multi-table sink write #7041

### PR DESCRIPTION
**Pull request by CosmosNi**
issue：#7041

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by CosmosNi** [comment]
@iture123 @TyrantLucifer @Hisoka-X PTAL,thanks.

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-elasticsearch-e2e/src/test/java/org/apache/seatunnel/e2e/connector/elasticsearch/ElasticsearchIT.java [hunk 1]
```diff
@@ -281,7 +304,80 @@ private List<String> readSinkData() throws InterruptedException {
                         "c_int",
                         "c_date",
                         "c_timestamp");
-        return getDocsWithTransformTimestamp(source, "st_index2");
+        return getDocsWithTransformTimestamp(source, index);
+    }
+
+    private List<String> readMultiSinkData5(String index) throws InterruptedException {
```
Comment (Hisoka-X) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-elasticsearch-e2e/src/test/java/org/apache/seatunnel/e2e/connector/elasticsearch/ElasticsearchIT.java]: what's different between `readMultiSinkData5` and `readMultiSinkData6`?

Comment (CosmosNi) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-elasticsearch-e2e/src/test/java/org/apache/seatunnel/e2e/connector/elasticsearch/ElasticsearchIT.java]: Schema is different. c_string

Comment (Hisoka-X) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-elasticsearch-e2e/src/test/java/org/apache/seatunnel/e2e/connector/elasticsearch/ElasticsearchIT.java]: How about like this?
```java
private List<String> readMultiSinkData(String index, List<String> source) throws
```

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @CosmosNi !

**Review by hailin0 — APPROVED** [review]

**Comment by bwcxyk** [comment]
Unable to set index and primary_keys for multiple tables.

**Pull request merged**